index.html

```diff
@@ -10318,19 +10302,23 @@ <h2>Taxonomy of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</ab
 			<li><a href="#attrs_liveregions">Live Region Attributes</a></li>
 			<li><a href="#attrs_dragdrop">Drag-and-Drop Attributes</a></li>
 			<li><a href="#attrs_relationships">Relationship Attributes</a></li>
+			<li><a href="#attrs_other">Other Attributes</a></li>
 		</ol>
+		<p class="note">Some <a>states</a> and [=ARIA/properties=] are applicable to all host language [=elements=] regardless of whether a <a>role</a> is applied. These global states and properties are supported by all roles and by all base markup elements unless otherwise prohibited. If a role prohibits use of any global states or properties, those states or properties are listed as prohibited in the characteristics table included in the section that defines the role.</p>
 		<section id="attrs_widgets">
 			<h3>Widget Attributes</h3>
 			<p>This section contains [=attributes=] specific to common user interface [=elements=] found on <abbr title="Graphical User Interface">GUI</abbr> systems or in rich internet applications which receive user input and process user actions. These attributes are used to support the <a href="#widget_roles">widget roles</a>.</p>
-			<ul>
+			<ul data-annotate-global-attrs>
 				<li><pref>aria-autocomplete</pref></li>
+				<li><pref>aria-braillelabel</pref></li>
 				<li><sref>aria-checked</sref></li>
 				<li><sref>aria-disabled</sref></li>
 				<li><pref>aria-errormessage</pref></li>
 				<li><sref>aria-expanded</sref></li>
 				<li><pref>aria-haspopup</pref></li>
 				<li><sref>aria-hidden</sref></li>
 				<li><sref>aria-invalid</sref></li>
+				<li><pref>aria-keyshortcuts</pref></li>
 				<li><pref>aria-label</pref></li>
 				<li><pref>aria-level</pref></li>
```

Comment:
aria-labelledby probably fits here too (aria-label is here so IMO they go together wherever they go)